docs/old-reference-guide/modules/events/pages/event-processors/subscribing.adoc

```diff
@@ -187,15 +187,15 @@ Events with the same value for these fields are passed in the same segment.
 
 The `PropertySequencingPolicy` requires 4 values in the `sequencingPolicyParameters` list.
 
-1. The serialization type for the events.
+1. The conversion type for the events.
```

Comment:
not sure if `conversion` makes sense in the context of this or if we should keep serializatoin